source/a11y-terms.ttl

```diff
@@ -0,0 +1,417 @@
+    vs:term_status "unstable" ;
+    skos:definition "The resource contains charts encoded in visual form."@en ;
+    skos:notation "chartOnVisual" .
+
+<accessMode/chemOnVisual> a :AccessMode ;
+    vs:term_status "unstable" ;
+    skos:definition "The resource contains chemical equations encoded in visual form."@en ;
+    skos:notation "chemOnVisual" .
+
+<accessMode/colorDependent> a :AccessMode ;
+    vs:term_status "unstable" ;
+    skos:definition "The resource contains information encoded in such that color perception is necessary."@en ;
+    skos:notation "colorDependent" .
+
+<accessMode/diagramOnVisual> a :AccessMode ;
+    vs:term_status "unstable" ;
+    skos:definition "The resource contains diagrams encoded in visual form."@en ;
+    skos:notation "diagramOnVisual" .
+
+<accessMode/mathOnVisual> a :AccessMode ;
+    vs:term_status "unstable" ;
+    skos:definition "The resource contains mathematical notations encoded in visual form."@en ;
+    skos:notation "mathOnVisual" .
+
+<accessMode/musicOnVisual> a :AccessMode ;
+    vs:term_status "unstable" ;
+    skos:definition "The resource contains music encoded in visual form."@en ;
+    skos:notation "musicOnVisual" .
+
+<accessMode/textOnVisual> a :AccessMode ;
+    vs:term_status "unstable" ;
+    skos:definition "The resource contains text encoded in visual form."@en ;
+    skos:notation "textOnVisual" .
+
+
+### 2. accessModeSufficient
+# A list of single or combined accessModes that are sufficient to understand 
+# all the intellectual content of a resource.
+
+<accessModeSufficient/auditory> a :AccessModeSufficient ;
+    vs:term_status "unstable" ;
+    skos:definition "Auditory perception is necessary to consume the information."@en ;
+    skos:notation "auditory" .
+
+<accessModeSufficient/tactile> a :AccessModeSufficient ;
+    vs:term_status "unstable" ;
+    skos:definition "Tactile perception is necessary to consume the information."@en ;
+    skos:notation "tactile" .
+
+<accessModeSufficient/textual> a :AccessModeSufficient ;
+    vs:term_status "unstable" ;
+    skos:definition "The ability to read textual content is necessary to consume the information."@en ;
+    #NOTE: Reading textual content does not require visual perception, as textual content can be rendered as audio using a text-to-speech capable device or assistive technology.
+    skos:notation "textual" .
+
+<accessModeSufficient/visual> a :AccessModeSufficient ;
+    vs:term_status "unstable" ;
+    skos:definition "Visual perception is necessary to consume the information."@en ;
+    skos:notation "visual" .
+
+#TODO: define sufficient logical combinations? the following two from MTM-examples:
+
+<accessModeSufficient/textualVisual> a :AccessModeSufficient ;
+    vs:term_status "unstable" ;
+    skos:notation "textual, visual" .
+
+<accessModeSufficient/auditoryVisual> a :AccessModeSufficient ;
+    vs:term_status "unstable" ;
+    skos:notation "auditory, visual" .
+
```

Comment:
Just add each base term individually *should* be enough? Comment out until we know if needed?

Reply:
Commented out ;)